plutus-pab-client/src/View/Pretty.purs

```diff
@@ -33,9 +32,6 @@ withHeading prefix content =
     , content
     ]
 
-instance prettyContractExe :: Pretty ContractExe where
-  pretty ((ContractExe { contractPath })) = text contractPath
```

Comment:
Mmm.... It's not really possible to create an instance of Pretty for Builtin, since it doesn't really have a data constructor with the contract.

@j-mueller How do you think this can be done? Do we need to use `ContractDef` to make that happen?

Reply:
I don't think we need a `prettyBuiltin` - the `Builtin` type has no values, so there isn't anything to pretty print. On the Haskell side we use `Builtin` with a type family. For example, the log messages for a `Builtin a` usually have `ContractDef a` as the concrete type. For `MarloweContract` this is just `MarloweContract` - so we'd need a `prettyMarloweContract`, but the instance needs to be written for each `a` individually, we can't provide a general one as part of `plutus-pab-client`.